src/Tooling/Bit.Tooling.Templates/TodoTemplate/Api/wwwroot/swagger/swagger-utils.js

```diff
@@ -14,17 +16,40 @@ const initLoginForm = () => {
 
     overrideSwaggerAuthorizeEvent(swagger);
     overrideSwaggerLogoutEvent(swagger);
+    loadLastSession(swagger);
     showLoginUI(swagger);
 }
 
+const loadLastSession = (swagger) => {
```

Comment:
it's not a session loading actually. it's just a swagger authorization process.
so please rename this method to something like `authorizeSwaggerWithLocalData`